__tests__/run.test.ts

```diff
@@ -0,0 +1,7 @@
+import {run} from '../src/login'
+
+describe('This is a placeholder for intial test cases, to be removed', () => {
+    test('Dummy test case', async () => {
+        await expect(run()).rejects.toThrow();
+    })
```

Comment:
Perhaps have a success test case? WDYT?

Reply:
That requires quite a few mocking. This is just the basic set-up. May be next code change can add a test cases and remove this one?

Reply:
Sure. But then the `jest` coverage would always show false results.. Do you plan on taking the tests part later down the line?

Reply:
Yeah. By this PR we will have a baseline coverage, any new code changes will break the coverage and we can start adding tests from there.